_posts/2016-02-24-drupal-6-part-4.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Drupal 6 retires today, now what? Part 4: Upgrade to D7 or D8?"
+date: 2016-02-24
+author: Chris Russo
+tags: drupal drupal8 drupal-planet
+summary: Which should you choose in upgrading your Drupal 6 site, Drupal 7 or 8?
+featured_image: "/assets/img/retirement.jpg"
+featured_image_alt: "a retirement sign at the beach"
+drupal_planet_summary:
+  Part 4 of a series investigating what options you have with your Drupal 6 site
+  as EOL has now come. We discuss whether Drupal 7 or Drupal 8 make sense for your
+  site.
+---
+
+This is part 4 of a series investigating what to do with your Drupal 6 site as
+EOL has now come.
+
+[Part 1 - overview](/2015/11/24/drupal-6-upgrade.html) \|
+[Part 2 - the risks](/2015/12/10/drupal-6-part-2.html) \|
+[Part 3 - the options](/2016/01/25/drupal-6-part-3.html)
+\| **Part 4 - Drupal 7 or Drupal 8?**
+
+*****
+
+
+<img src="/assets/img/retirement.jpg" alt="a retirement sign at the beach">
+
+### Should I upgrade my Drupal 6 site to 7 or 8?
+
+Today Drupal 6 reaches retirement, or end of life (EOL) which means the Drupal
+community and security team will no longer officially support active development, 
+security and bug fixing for the platform. We instead must focus our resources on 
+maintaining Drupal 7 and 8. 
+
+Of course, since Drupal is open source software, even though EOL for Drupal 6
+_is_ today, some organizations will continue to operate their live websites on 
+Drupal 6 for some time. Upgrading to a current, supported version of Drupal is
+recommended, but what makes for the right choice for your organization, Drupal 7 
```

Comment:
> what makes for the right choice for your organization, Drupal 7 or Drupal 8, is conditional.

Maybe change this to "but the right choice for your organization, Drupal 7 or Drupal 8, is conditional."